src/main/java/pl/pelikan/weatherinthecity/NoCityFoundException.java

```diff
@@ -0,0 +1,7 @@
+package pl.pelikan.weatherinthecity;
+
+public class NoCityFoundException extends RuntimeException {
```

Comment:
Jak dla mnie to nie powinien być RuntimeException, tylko zwykły (Checked exception). Wiem, że mniej przyjemnie się go obsługuje w kodzie, ale powinniśmy wiedzieć kiedy go wyrzucamy